M2/Macaulay2/packages/Macaulay2Doc/functions/gb-doc.m2

```diff
@@ -326,3 +326,388 @@ document {
     "The option ", TO "ChangeMatrix", " can be used with ", TO "gb",
     " to enable the computation of the change of basis matrix."
 }
+
+-- Tests for different variants
+
+TEST /// -- basics of Algorithm => LinearAlgebra
```

Comment:
These tests don't get run. Either move them to `EngineTests` or the `tests` directory.

Reply:
I moved them to EngineTests/GB.Test.LinearAlgebra.m2 to get the tests running.